packages/gestalt/src/IconButtonFloating.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+import { forwardRef, type Node, useImperativeHandle, useRef } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import icons from './icons/index.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+
+type Props = {|
+  /**
+   * String that clients such as VoiceOver will read to describe the icon button. Always localize the label. See [Accessibility section](https://gestalt.pinterest.systems/web/modalalert#Accessibility) for more info.
+   */
+  accessibilityLabel: string,
+  icon?: $Keys<typeof icons>,
+  onClick?: AbstractEventHandler<
```

Comment:
yeah, look here to get an orientation packages/gestalt/src/SideNavigationTopItem.js